internal/plugin/slo/contrib/error_budget_exhausted_alert_v1/plugin.go

```diff
@@ -0,0 +1,115 @@
+package plugin
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"sort"
+	"strings"
+
+	"github.com/prometheus/common/model"
+	"github.com/prometheus/prometheus/model/rulefmt"
+
+	pluginslov1 "github.com/slok/sloth/pkg/prometheus/plugin/slo/v1"
+)
+
+const (
+	PluginVersion = "prometheus/slo/v1"
+	PluginID      = "sloth.dev/contrib/error_budget_exhausted_alert/v1"
+)
+
+type Config struct {
+	Threshold      float64           `json:"threshold"`                 // default 0, fully exhausted
+	For            string            `json:"for"`                       // default 5m
```

Comment:
Prom `model.Duration` type already has [automatic JSON management](https://pkg.go.dev/github.com/prometheus/common@v0.66.1/model#Duration.UnmarshalJSON). So you can simplify the config by removing `ForDuration` field and avoid the parsing.

Reply:
good call, updated [`eb5e619` (#670)](https://github.com/slok/sloth/pull/670/commits/eb5e619067f5bf289cf175baf822af7e22c5b58d)